### ISSUE TITLE
Feature Request : Quick Glance Statistics of devices statuses

### ISSUE THREAD

**Issue by jaimzj** [traccar/traccar#4269]
The idea is to have a panel displaying quick stats of all devices under a user account (example of it has been attached to give an idea).

Possibly user selectable or configurable Attributes, of which the total count will be displayed somewhere on the UI. (Example, total number of devices with Ignition On, or Total number of devices currently overspeeding, or total number of devices offline at the moment) etc.

Pain Area : At the moment if a user needs to check how many of their devices have ignition status on, or are over speeding, or are stopped and stationary, they need to click on each device/marker to be able to see each devices state individually.

Use Case : Users can upon login get an immediate glance of their overall all devices status in simple number based statistics based on device status, or an attributes value (count).

![image](https://user-images.githubusercontent.com/1138485/55660884-8a837680-57d6-11e9-99f1-129dd9a9d470.png)

**Comment by tananaev** [comment]
It would also be useful to have total number of devices as well.

**Comment by kirantpatil** [comment]
How about "Total Trips" on device status ?

**Comment by jaimzj** [comment]
@kirantpatil That might possibly be a lot of work I feel, however just showing the total count of devices based on its attributes state would be really useful to almost all users.

Hopefully if a lot of users up-vote this request it might catch some traction in terms of having this feature implemented.  And maybe that could form the basis for additional values like you suggested like total trips, total distance etc.

**Comment by gpshex007** [comment]
I am looking for something like this, and there seem to be quite some demand for this request based on up votes @tananaev could this request be considered for the upcoming release?

**Comment by tananaev** [comment]
I don't think it makes sense to implement it in the current version of the web app. It would only make sense to be implement in the new web app, but it's still in a very early stage.

**Comment by jaimzj** [comment]
@tananaev noticed much progress on the new web app :) So just wanted to bring this topic back in. Hoping you or someone might be able to add this in as a new feature :)

**Comment by tananaev** [comment]
As I mentioned several times, the priority is to migrate existing functionality first. But you are welcome to contribute.
